lib/util.js

```diff
@@ -734,11 +726,19 @@ function formatMap(ctx, value, recurseTimes, visibleKeys, keys) {
   return output;
 }
 
-function formatCollectionIterator(ctx, value, recurseTimes, visibleKeys, keys) {
-  ensureDebugIsInitialized();
-  const mirror = Debug.MakeMirror(value, true);
+function formatMapIterator(ctx, value, recurseTimes, visibleKeys, keys) {
+  var nextRecurseTimes = recurseTimes === null ? null : recurseTimes - 1;
+  var vals = previewMapIterator(value, 100);
```

Comment:
There is a minor change in behavior here in that the preview is now limited to 100 elements.  Before, it would show them all but that is (IMO) rather annoying with large maps and sets (and it's basically equivalent to printing the map or set minus the already iterated over elements.)

I'm planning to follow up with a PR that prints a nice "..." if there are more elements but I can restore the old behavior if people prefer that.